core/server/master/src/main/java/alluxio/master/meta/checkconf/ServerConfigurationChecker.java

```diff
@@ -36,7 +36,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(ServerConfigurationChecker.class);
   private static final int LOG_CONF_SIZE = 5;
   private static final String CONSISTENT_CONFIGURATION_INFO
-      = "All server-side configuration are consistent.";
+      = "All server-side configuration is consistent.";
```

Comment:
`configurations are` or `The server-side configuration is`?

Reply:
updated to configurations are, thanks